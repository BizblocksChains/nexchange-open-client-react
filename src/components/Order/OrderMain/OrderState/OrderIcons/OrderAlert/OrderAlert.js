import React from 'react';
import styles from './OrderAlert.scss';

const OrderAlert = () => {
  return (
    <div className={styles.icon}>
      <svg
        version="1.1"
        id="Layer_1"
        xmlns="http://www.w3.org/2000/svg"
        x="0px"
        y="0px"
        viewBox="0 0 39.7 35.2"
        style={{ enableBackground: 'new 0 0 39.7 35.2' }}
      >
        <g>
          <g>
            <path
              d="M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6
			s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33
			c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z"
            />
          </g>
          <g>
            <path
              d="M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5
			c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4
			C21.3,25.7,21.4,26.1,21.4,26.6z"
            />
          </g>
        </g>
      </svg>
    </div>
  );
};

export default OrderAlert;
