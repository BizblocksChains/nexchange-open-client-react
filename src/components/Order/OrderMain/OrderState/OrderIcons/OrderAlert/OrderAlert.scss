.icon {
  svg {
    width: 40px;
    height: 40px;
    margin: 10px 0 -11px 0;
  }

  svg path,
  svg rect {
    fill: #eda900;
  }
}
