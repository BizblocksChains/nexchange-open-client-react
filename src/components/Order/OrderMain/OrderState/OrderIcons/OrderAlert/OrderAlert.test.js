import React from 'react';
import { shallow } from 'enzyme';
import OrderAlert from './OrderAlert';

describe('OrderAlert', () => {
  it('renders correctly', () => {
    const wrapShallow = shallow(<OrderAlert />);
    expect(wrapShallow).toMatchSnapshot();
  });
});
