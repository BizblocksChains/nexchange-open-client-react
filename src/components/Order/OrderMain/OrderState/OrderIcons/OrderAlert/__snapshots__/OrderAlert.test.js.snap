// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderAlert renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OrderAlert />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <svg
        id="Layer_1"
        style={
                Object {
                        "enableBackground": "new 0 0 39.7 35.2",
                      }
        }
        version="1.1"
        viewBox="0 0 39.7 35.2"
        x="0px"
        xmlns="http://www.w3.org/2000/svg"
        y="0px"
>
        <g>
                <g>
                        <path
                                d="M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z"
                        />
                </g>
                <g>
                        <path
                                d="M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z"
                        />
                </g>
        </g>
</svg>,
      "className": "icon",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <g>
          <g>
                    <path
                              d="M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z"
                    />
          </g>
          <g>
                    <path
                              d="M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z"
                    />
          </g>
</g>,
        "id": "Layer_1",
        "style": Object {
          "enableBackground": "new 0 0 39.7 35.2",
        },
        "version": "1.1",
        "viewBox": "0 0 39.7 35.2",
        "x": "0px",
        "xmlns": "http://www.w3.org/2000/svg",
        "y": "0px",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <g>
              <path
                            d="M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z"
              />
</g>,
            <g>
              <path
                            d="M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z"
              />
</g>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <path
                d="M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z"
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            "type": "g",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <path
                d="M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z"
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            "type": "g",
          },
        ],
        "type": "g",
      },
      "type": "svg",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <svg
          id="Layer_1"
          style={
                    Object {
                              "enableBackground": "new 0 0 39.7 35.2",
                            }
          }
          version="1.1"
          viewBox="0 0 39.7 35.2"
          x="0px"
          xmlns="http://www.w3.org/2000/svg"
          y="0px"
>
          <g>
                    <g>
                              <path
                                        d="M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z"
                              />
                    </g>
                    <g>
                              <path
                                        d="M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z"
                              />
                    </g>
          </g>
</svg>,
        "className": "icon",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <g>
            <g>
                        <path
                                    d="M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z"
                        />
            </g>
            <g>
                        <path
                                    d="M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z"
                        />
            </g>
</g>,
          "id": "Layer_1",
          "style": Object {
            "enableBackground": "new 0 0 39.7 35.2",
          },
          "version": "1.1",
          "viewBox": "0 0 39.7 35.2",
          "x": "0px",
          "xmlns": "http://www.w3.org/2000/svg",
          "y": "0px",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <g>
                <path
                                d="M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z"
                />
</g>,
              <g>
                <path
                                d="M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z"
                />
</g>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <path
                  d="M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z"
/>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "d": "M36.9,35.2h-34c-1.2,0-2.1-0.5-2.5-1.3c-0.5-0.8-0.4-1.8,0.2-2.8l17-29.5c0.6-1,1.4-1.6,2.4-1.6 s1.8,0.6,2.4,1.6h0l17,29.5c0.6,1,0.6,2,0.2,2.8S38,35.2,36.9,35.2z M19.9,2c-0.1,0-0.4,0.1-0.6,0.6l-17,29.5C2,32.5,2,32.8,2,33 c0.1,0.1,0.3,0.3,0.8,0.3h34c0.5,0,0.7-0.1,0.8-0.3c0.1-0.1,0.1-0.4-0.2-0.8l-17-29.5h0C20.2,2.1,20,2,19.9,2z",
                },
                "ref": null,
                "rendered": null,
                "type": "path",
              },
              "type": "g",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <path
                  d="M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z"
/>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "d": "M18.8,24l-0.5-9.9h3.1L20.9,24H18.8z M21.4,26.6c0,0.4-0.1,0.7-0.4,1c-0.3,0.3-0.6,0.5-1.1,0.5 c-0.5,0-0.9-0.1-1.1-0.4s-0.4-0.6-0.4-1.1c0-0.5,0.1-0.9,0.4-1.1c0.3-0.3,0.6-0.4,1.1-0.4c0.5,0,0.8,0.1,1.1,0.4 C21.3,25.7,21.4,26.1,21.4,26.6z",
                },
                "ref": null,
                "rendered": null,
                "type": "path",
              },
              "type": "g",
            },
          ],
          "type": "g",
        },
        "type": "svg",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
